mangaki/mangaki/models.py

```diff
@@ -248,6 +248,31 @@ def update_tags(self, anidb_tags):
 
     # Should add an update_creators method similar to update_tags
 
+    def is_nsfw_based_on_tags(self, anidb_tags):
+        # Might want to move this somewhere else ...
+        potentially_nsfw_tags = ['nudity', 'ecchi', 'pantsu', 'breasts', 'sex',
```

Comment:
Add an explicit `FIXME` for items which we should take action on.
I believe that at some point, we should have a `constants` module or something like this, but let's do it in another PR for refactor/cleanup.